spec/models/book_spec.rb

```diff
@@ -5,6 +5,9 @@
     it {should validate_presence_of :title}
     it {should validate_presence_of :page_count}
     it {should validate_presence_of :year_published}
+    it {should validate_uniqueness_of :title}
+    it {should validate_numericality_of :year_published}
```

Comment:
We need to add tests of the other validations you added for both year_published and page_count.

Reply:
Done, added on branch.